src/lib.rs

```diff
@@ -113,7 +113,7 @@ pub mod contract {
         pub target_sqrt_price: SqrtPrice,
         pub fee: TokenAmount,
         pub pool: Pool,
-        pub ticks: Vec<Tick>,
+        pub ticks: Vec<(Tick, bool)>,
```

Comment:
compute swap step should return only crossed ticks. no crossing ticks don't matter